README.md

```diff
@@ -1,4 +1,7 @@
 # hmpps-template-typescript
+[![repo standards badge](https://img.shields.io/badge/dynamic/json?color=blue&style=for-the-badge&logo=github&label=MoJ%20Compliant&query=%24.result&url=https%3A%2F%2Foperations-engineering-reports.cloud-platform.service.justice.gov.uk%2Fapi%2Fv1%2Fcompliant_public_repositories%2Fhmpps-template-typescript)](https://operations-engineering-reports.cloud-platform.service.justice.gov.uk/public-github-repositories.html#hmpps-template-typescript "Link to report")
```

Comment:
Not a big thing but in https://github.com/ministryofjustice/hmpps-incentives-api/pull/63 I used `style=flat` so the badge was the same size & shape as others

Reply:
ah that looks better! @neilmendum are you ok to merge this if you're happy with it?